metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/QueueConsumer.java

```diff
@@ -214,6 +214,8 @@ private void checkCollectedWorkflowExecution(
    */
   public void close() {
     threadPool.shutdown();
+    //Interrupt running threads
+    threadPool.shutdownNow();
```

Comment:
I think this is overkill. The first call does not block, so if you're going to interrupt everything, might as well do it straight away.

Reply:
Wouldn't matter much, I guess we can avoid the first call